### PR TITLE
sample function added in server.py

### PR DESCRIPTION
**Pull request by Arumugamvasu**
sample function added in server.py

### CODE REVIEW

File: User.py [hunk 1]
```diff
@@ -9,4 +9,9 @@ class user():
     def __init__ (self,_id,username,password):
         self.id = _id
         self.username = username
-        self.password = password
\ No newline at end of file
+        self.password = password
+        
+    def sample_fun(self):
+        print(self.id)
```
Comment (sundararumugam) [inline User.py:15]: you need to print with details using format string

File: smple.txt [hunk 2]
```diff
@@ -1 +1,3 @@
-this is first commit for all to arumugamece879 mail like hp host 
\ No newline at end of file
+this is first commit for all to arumugamece879 mail like hp host 
+
+i have add new line here
```
Comment (sundararumugam) [inline smple.txt:3]: give some data here

File: User.py (hunk 1 above)
Comment (Arumugamvasu) [reply User.py:15]: ok i changed

**Review by sundararumugam — CHANGES_REQUESTED** [review]

**Comment by sundararumugam** [comment]
ok completed

**Comment by sundararumugam** [comment]
i changes is completed

**Comment by sundararumugam** [comment]
ok

**Pull request closed without merging**